src/controllers/ChatsController.cpp

```diff
@@ -139,6 +139,22 @@ void ChatsController::closeChat()
     setCurrentChat({});
 }
 
+void ChatsController::addParticipant(const UserId &userId)
+{
+    // addParticipant
```

Comment:
Comment doesn't introduce new information, remove.

For such situation you can leave TODO comment:
`TODO: implement` or `TODO(d.zezik): implement`

Reply:
+